docs/vm-setup.md

```diff
@@ -12,13 +12,16 @@
 2. Install a Windows VM as follows:
     1. Run "Hyper-V Manager".
     2. Select the current machine in the left pane.
-    3. Click the "Quick Create..." action in the rightmost pane.
-    4. When the Create Virtual Machine dialog appears, select "Windows 10 dev environment".
-    5. Click the "Create Virtual Machine" button.
-    6. Once that is complete click the "Edit Settings" button.
-    7. Select security, clear the "Enable Secure Boot" checkbox, and click OK. (This is a prerequisite for
+    3. Click the "Quick Create..." action in the rightmost pane. When the Create Virtual Machine dialog appears:
+       1. Select "Windows 11 dev environment".
+       2. Alternatively, click the "Local installation source" to install Windows OS of your choice from an .iso or a .vhd file.
```

Comment:
```suggestion
       2. Alternatively, click the "Local installation source" to install the Windows OS of your choice from an .iso or a .vhd file.
```

Reply:
I'll also observe that this doc allows .iso or .vhd whereas SelfHostedRunnerSetup.md only allows vhd.  I'm curious why the difference exists.